tests/scripts/thread-cert/Cert_8_1_02_Commissioning.py

```diff
@@ -40,11 +42,13 @@ class Cert_8_1_02_Commissioning(thread_cert.TestCase):
 
     TOPOLOGY = {
         COMMISSIONER: {
-            'masterkey': 'deadbeefdeadbeefdeadbeefdeadbeef',
+            'name': 'COMMISSIONER',
+            'masterkey': '00112233445566778899aabbccddeeff',
```

Comment:
why change the masterkey?
Isn't this test intended to detect the use of an incorrect PSKd by a Joiner?

Reply:
change masterkey to align with pktverify consts, change bake to check different PSKd

Reply:
I am not sure why the original test used different masterkey for Commsisioner. Do you have any insight on this? @FiveDimensions

Reply:
I think it may refer to the previous testplan. In latest test plan, it doesn't mention the masterkey but only needs incorrect PSKd for Joiner. I changed the Joiner masterkey to  deadbeefdeadbeefdeadbeefdeadbeef (different master-key and pskd), verification works as well.  Should we use the same master-key? @simonlingoogle & @jwhui

Reply:
I am fine with the current change.